xain_fl/__main__.py

```diff
@@ -0,0 +1,41 @@
+"""This module is the entrypoint to start a new coordinator instance.
+
+"""
+import sys
+
+import numpy as np
+
+from xain_fl.config import Config, InvalidConfig, get_cmd_parameters
+from xain_fl.coordinator.coordinator import Coordinator
+from xain_fl.coordinator.store import Store
+from xain_fl.logger import StructLogger, get_logger
+from xain_fl.serve import serve
+
+logger: StructLogger = get_logger(__name__)
+
+
+def main():
+    """Start a coordinator instance
+    """
+
+    args = get_cmd_parameters()
+    try:
+        config = Config.load(args.config)
+    except InvalidConfig as err:
+        logger.error(f"Invalid config: {err}")
```

Comment:
As we use structlog, would be nice to have structure here, and one of the best practices is not to use f-strings in logs: [More info about issue with f-strings](https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs/54368086)
Structlog log entry looks like:
```suggestion
        logger.error("Invalid config", error=repr(err))
```
repr will give us type of the error, we can also log any other specific information as a key-value pair.